### PR TITLE
apply uniq to attributes in the sequelizeConnection

### PR DESCRIPTION
**Pull request by bringking**
This PR should address #119 and #155, applies _.uniq to options.attributes in the sequelizeConnection constructor and implements a test in `relay/connection.test.js` to verify the proper functionality.

### CODE REVIEW

File: test/integration/relay/connection.test.js [hunk 1]
```diff
@@ -284,6 +331,112 @@ if (helper.sequelize.dialect.name === 'postgres') {
...
+            }
+          }),
+          edgeFields: {
+            isOwner: {
+              type: GraphQLBoolean,
+              resolve: function (edge) {
+                return edge.node.ownerId === edge.source.id;
+              }
+            }
+          }
```
Comment (mickhansen) [inline test/integration/relay/connection.test.js]: edgeFields and orderBy are not relevant here are they?

File: test/integration/relay/connection.test.js [hunk 2]
```diff
@@ -284,6 +331,112 @@ if (helper.sequelize.dialect.name === 'postgres') {
...
+          fields: {
+            ...attributeFields(this.User),
+            id: globalIdField(this.User.name),
+            tasks: {
+              type: this.userTaskConnection.connectionType,
+              args: {
+                ...this.userTaskConnection.connectionArgs,
+                completed: {
+                  type: GraphQLBoolean
+                }
```
Comment (mickhansen) [inline test/integration/relay/connection.test.js]: Isn't relevant either AFAIR (best to keep the test as small as possible while showing the issue)

File: test/integration/relay/connection.test.js (hunk 1 above)
Comment (bringking) [inline test/integration/relay/connection.test.js]: Right, I just cleaned them up.

**Comment by mickhansen** [comment]
Thanks again @bringking

**Pull request merged**